snippet-extractor-output/snippets.yaml

```diff
@@ -11153,6 +11153,43 @@
         for (var i = 0; i < properties.items.length; i++)
             console.log("Property Name:" + properties.items[i].key + "; Type=" + properties.items[i].type + "; Property Value=" + properties.items[i].value);
     });
+'Word.Document#changeTrackingMode:member':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/99-preview-apis/manage-change-tracking.yaml
+
+    await Word.run(async (context) => {
+      const document = context.document;
+      document.load("changeTrackingMode");
+      await context.sync();
+
+      if (document.changeTrackingMode === Word.ChangeTrackingMode.trackMineOnly) {
+        console.log("ChangeTracking is on for just mine.");
+      } else if (document.changeTrackingMode === Word.ChangeTrackingMode.trackAll) {
+        console.log("ChangeTracking is on for everyone.");
+      } else {
+        console.log("ChangeTracking is off.");
+      }
+    });
+  - >-
```

Comment:
I think having just one of these snippets shows the API sufficiently. I like the `get` version because it doesn't have jQuery.